test/btree.js

```diff
@@ -0,0 +1,207 @@
+    .insert(25,  { city: 'Washington' })
+    .insert(62,  { city: 'Lima'       })
+    .insert(64,  { city: 'Beijing'    });
+
+  let res = bTree.get(40);
+  test.strictSame(res.city, 'Abu Dhabi');
+  res = bTree.get(30);
+  test.strictSame(res.city, 'Ankara');
+  res = bTree.get(80);
+  test.strictSame(res.city, 'Jerusalem');
+  res = bTree.get(20);
+  test.strictSame(res.city, 'Seoul');
+  res = bTree.get(-100);
+  test.strictSame(res, null);
+  res = bTree.get(25);
+  test.strictSame(res.city, 'Washington');
+  test.end();
+});
+
+api.metatests.test('iterator / number value', (test) => {
+  const bTree = new BTree(2);
+  bTree
+    .insert(40,  'G')
+    .insert(30,  'F')
+    .insert(72,  'M')
+    .insert(24,  'E')
+    .insert(20,  'D')
+    .insert(18,  'B')
+    .insert(50,  'I')
+    .insert(60,  'L')
+    .insert(90,  'N')
+    .insert(45,  'H')
+    .insert(55,  'K')
+    .insert(52,  'J')
+    .insert(100, 'O')
+    .insert(120, 'P')
+    .insert(130, 'Q')
+    .insert(140, 'R')
+    .insert(19,  'C')
+    .insert(-20, 'A');
+
+  let res = [...bTree.iterator()].join(''); // All nodes
+  test.strictSame(res, 'ABCDEFGHIJKLMNOPQR');
+
+  res = [...bTree.iterator(undefined)].join(''); // All nodes
+  test.strictSame(res, 'ABCDEFGHIJKLMNOPQR');
+
+  res = [...bTree.iterator(undefined, undefined)].join(''); // All nodes
+  test.strictSame(res, 'ABCDEFGHIJKLMNOPQR');
+
+  res = [...bTree.iterator(undefined, 10000)].join(''); // All nodes
+  test.strictSame(res, 'ABCDEFGHIJKLMNOPQR');
+
+  res = [...bTree.iterator(-10000, undefined)].join(''); // All nodes
+  test.strictSame(res, 'ABCDEFGHIJKLMNOPQR');
+
+  res = [...bTree.iterator(100, 0)].join(''); // Empty
+  test.strictSame(res, '');
+
+  res = [...bTree.iterator(-200, -100)].join(''); // Empty
+  test.strictSame(res, '');
+
+  res = [...bTree.iterator(20, 50)].join('');
+  test.strictSame(res, 'EFGH');
+
+  res = [...bTree.iterator(100, 1000)].join('');
+  test.strictSame(res, 'PQR');
+
+  res = [...bTree.iterator(10, 99)].join('');
+  test.strictSame(res, 'BCDEFGHIJKLMN');
+
+  res = [...bTree.iterator(10, undefined)].join('');
+  test.strictSame(res, 'BCDEFGHIJKLMNOPQR');
+
+  res = [...bTree.iterator(126, 128)].join('');
+  test.strictSame(res, '');
+  test.end();
+});
+
+api.metatests.test('remove / number value', (test) => {
+  const bTree = new BTree(2);
+
+  bTree
+    .insert(40,  { city: 'Abu Dhabi'  })
+    .insert(30,  { city: 'Ankara'     })
+    .insert(80,  { city: 'Jerusalem'  })
+    .insert(20,  { city: 'Seoul'      })
+    .insert(100, { city: 'Kiev'       })
+    .insert(110, { city: 'Luxembourg' })
+    .insert(18,  { city: 'Minsk'      })
+    .insert(135, { city: 'Oslo'       })
+    .insert(50,  { city: 'Stockholm'  })
+    .insert(25,  { city: 'Washington' })
+    .insert(62,  { city: 'Lima'       })
+    .insert(64,  { city: 'Beijing'    });
+
+  bTree.remove(40);
+  let res = bTree.get(40); // Remove root
+  test.strictSame(res, null);
```

Comment:
If must be not null, but `undefined` here. That's what `undefined` means - no such value.